pepper-apis/src/main/resources/db-changes/schema/add-last-visited-activity-section-column-to-activity-instance.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="20200528-add-last-visited-activity-section-to-activity-instance" author="sergei-poliakov">
+
+        <addColumn tableName="activity_instance">
+            <column name="last_visited_section" type="int" defaultValue="0">
+                <constraints nullable="false"/>
```

Comment:
We need to nail down semantics of 0. Never visited or visited first section only?

Reply:
in terms of user experience, it doesn't matter – he'll see the first section anyway